datasets/modeled/industry/dataset.json5

```diff
@@ -0,0 +1,118 @@
+{
+  dataset_id: "decarb_2023_industry",
+  dataset_type: "modeled",
+  data_source: "ILSST_IGATE-E_GCAM",
+  sector_description: "industrial_manufacturing",
+  description: "Load shapes modeled for the manufacturing sector created using LBNL Industrial Load Shape Simulation Tool, national facility demand estimates from IGATE-E, and forecasts from GCAM.",
+  origin_creator: "Sarah Smith",
+  origin_organization: "LBL",
+  origin_contributors: [
+    "Samanvitha Murthy",
+    "Ikenna Okeke",
+    "Colin McMillian",
+  ],
+  origin_project: "DECARB Task 6 Year 1",
+  origin_date: "2024-02-06",
+  origin_version: "0.1.0",
+  source: "LBL SIELS (Simulator for Industrial Electricity Load Shapes)",
```

Comment:
This seems out of sync with the tool name listed above ("LBNL Industrial Load Shape Simulation Tool") @sjsmith11 

Maybe it would make sense here just to list organizations that contributed, e.g., "LBNL, ORNL, NREL".